src/main/scala/io/sdkman/changelogs/java/MicrosoftOpenJDKMigrations.scala

```diff
@@ -0,0 +1,34 @@
+package io.sdkman.changelogs.java
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+import io.sdkman.changelogs.{Linux64, MacOSX, Microsoft, Version, Windows}
+
+@ChangeLog(order = "047")
+class MicrosoftOpenJdkMigrations {
+
+  @ChangeSet(
+    order = "003",
```

Comment:
`MicrosoftOpenJdkMigrations` already existed but was dropped few commits ago. I have fix the id suffix to 003 because it was the next one when the file was deleted.